DOCS/man/mpv.rst

```diff
@@ -1200,9 +1200,17 @@ PROTOCOLS
 
     Digital TV via DVB. (Linux only.)
 
-``mf://[filemask|@listfile]`` ``--mf-...``
-
-    Play a series of images as video.
+``mf://[@listfile|filemask|glob|printf-format]`` ``--mf-...``
+
+    Play a series of images as a video. If the URL begins with ``@``, it is
+    interpreted as a sequence of paths separated by newlines. If the URL
+    contains ``,``, it is interpreted as a sequence of paths separated by ``,``.
+    If the URL doesn't contain ``%`` and if on POSIX platforms, the URL is
+    interpreted as a glob, and ``*`` is automatically appended if it was not
+    specified. Otherwise, the printf sequences ``%[.][NUM]d``, where ``NUM`` is
+    one, two, or three decimal digits, and ``%%`` and are interpreted. For
+    example, ``mf://image-%d.jpg`` plays files like ``image-1.jpg``,
+    ``image-2.jpg`` and ``image-10.jpg``.
```

Comment:
it doesn't actually
there are more than 5 missing between 2 and 10 so it would only play the first two